jkube-kit/jkube-kit-micronaut/src/main/java/org/eclipse/jkube/micronaut/enricher/MicronautHealthCheckEnricher.java

```diff
@@ -85,10 +85,11 @@ private Probe buildProbe(Integer initialDelaySeconds, Integer periodSeconds){
       return null;
     }
 
-    final String firstImagePort = getImages().stream().findFirst()
+    String port = getImages().stream().findFirst()
```

Comment:
Could you please revert it to the previous name? I think it was more accurate than the renamed version.
```suggestion
    final String firstImagePort = getImages().stream().findFirst()
```